### PR TITLE
Add option to ignore hit counts

### LINKED ISSUE #63: Add option to ignore hit counts
Hit counts are not perfect, so making the diff could be a problem.
We could add an option to the diffing function to always consider hit counts > 0 as being 1.

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #63

### CODE REVIEW

File: test/test_diff.py [hunk 1]
```diff
@@ -689,3 +689,115 @@ def test_compare_reports_no_diff():
...
+                    {
+                        'name': 'func2',
+                        'start': 64,
+                        'exec': True
+                    }
+                ],
+                'branches': [],
+                'name': 'obj-firefox/dist/include/mozilla/dom/DOMRectListBinding.h',
+                'coverage': [
+                    None,
```
Comment (marco-c) [inline test/test_diff.py]: Could you also add a test that returns an array that is not completely filled with None and 0?
E.g. `[ None, 2, None, 1, 0 ]` for baseline and `[ None, 2, None, 5, 6 ]` for report.

Comment (rhcu) [reply test/test_diff.py]: Oops. It seems that I forgot to set diff as 1 if hit counts > 0.

Comment (marco-c) [reply test/test_diff.py]: Right! This shows why unit tests are important :smile:

File: diff.py [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 
-def diff_line(i, j):
+def diff_line(i, j, ignore_hits):
     if j is None:
         return None
     elif i is None:
         return j
-    elif j > 0 and j >= i:
+    elif j > 0 and j >= i and ignore_hits is False:
         return j - i
```
Comment (marco-c) [inline diff.py]: You can have a branch here instead:
```
elif j > 0 and j > i:
    if ignore_hits:
        return 1
    else:
        return j - i
else:
    return 0
```

Comment (rhcu) [reply diff.py]: Seems that with such condition there won't be 0 if in baseline coverage is 1 and in the report is 5, for example.
